db/migrations/versions/592143492996_add_scores_justifications_table.py

```diff
@@ -0,0 +1,58 @@
+"""add scores justifications table
+
+Revision ID: 592143492996
+Revises: 4dd55a3dc51e
+Create Date: 2022-07-05 16:02:00.331445
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy_utils.types import UUIDType
+
+
+# revision identifiers, used by Alembic.
+revision = "592143492996"
+down_revision = "4dd55a3dc51e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "sub_criteria",
+        sa.Column("sub_criteria_id", UUIDType(binary=False), nullable=False),
+        sa.Column("round_id", sa.Text(), nullable=True),
+        sa.Column("criteria_id", sa.Text(), nullable=True),
+        sa.Column("sub_criteria_title", sa.Text(), nullable=True),
+        sa.PrimaryKeyConstraint("sub_criteria_id"),
+    )
+    op.create_table(
+        "scores_justifications",
+        sa.Column(
+            "scores_justifications_id", UUIDType(binary=False), nullable=False
+        ),
+        sa.Column("created_at", sa.DateTime(), nullable=True),
+        sa.Column("assessment_id", sa.Text(), nullable=True),
+        sa.Column("person_id", sa.Text(), nullable=True),
+        sa.Column("sub_criteria_id", sa.Text(), nullable=True),
```

Comment:
For these short text fields can we use `String` type with a limit of 255 length? Else we will be storing them inefficiently in the database.

Reply:
👍🏾